finance-tracker/app/controllers/user_stocks_controller.rb

```diff
@@ -1,2 +1,12 @@
 class UserStocksController < ApplicationController
+  def create
+    stock = Stock.check_db(params[:ticker])
```

Comment:
Why need this method? Why not straight do the query here itself?

Reply:
and, instead of do `where(..).first`, you can just do `find_by(...)`

Reply:
I figure it is because of the DRY code. So the method can be used in different places without repeating the same lines of code.

Reply:
its just:
```
Stock.find_by(ticker: ticker_symbol)
```
I think that is dry enough.